app/src/components/market/sections/market_sell/scalar_market_sell.tsx

```diff
@@ -187,7 +252,7 @@ export const ScalarMarketSell = (props: Props) => {
       setAmountShares(null)
       setAmountSharesToDisplay('')
```

Comment:
I think we might have to zero out `displaySellShares` here after a sell.

Reply:
Yes good catch. Fixed